### PR TITLE
[FrameworkBundle] Ignore backslashes in service ids when using debug:container and debug:autowiring

### PR DESCRIPTION
**Pull request by respinoza**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #28143 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]
Thanks for submitting!

**Comment by respinoza** [comment]
@nicolas-grekas do we need a doc PR for this?
Maybe https://symfony.com/doc/current/service_container/debug.html ?

**Comment by nicolas-grekas** [comment]
I'm not sure actually.

File: src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ CHANGELOG
  * Added new "auto" mode for `framework.session.cookie_secure` to turn it on when HTTPS is used
  * Removed the `framework.messenger.encoder` and `framework.messenger.decoder` options. Use the `framework.messenger.serializer.id` option to replace the Messenger serializer. 
  * Deprecated the `ContainerAwareCommand` class in favor of `Symfony\Component\Console\Command\Command`
+ * Add the ability to ignore backslashes in service ids when using `debug:container` and `debug:autowiring`
```
Comment (chalasr) [inline src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md]: `Added` :)

**Review by chalasr — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @respinoza.

**Pull request merged**